aggregator/src/aggregator.rs

```diff
@@ -772,6 +772,31 @@ impl<C: Clock> Aggregator<C> {
             .await
     }
 
+    /// Handle an aggregate share deletion request. Only supported by the helper.
+    async fn handle_delete_aggregate_share(
+        &self,
+        task_id: &TaskId,
+        aggregate_share_id: &AggregateShareId,
+        auth_token: Option<AuthenticationToken>,
+        taskprov_task_config: Option<&TaskConfig>,
+    ) -> Result<(), Error> {
+        let task_aggregator = self
```

Comment:
This should check that we are acting as the helper in this task, akin to the check in [`handle_delete_collection_job`](https://github.com/divviup/janus/blob/f0b9435b764042254252607796861e6125924705/aggregator/src/aggregator.rs#L659)